migrations/20160901_remove_null_notifications.js

```diff
@@ -0,0 +1,13 @@
+var _ = require('lodash');
+
+db.users.find({}).forEach(function(user){
+  var newNewMessages = {};
+
+  _.each(user.newMessages, function(val, key){
+    if(key != "undefined" && val!=null){
+      newNewMessages[key] = val;
+    }
+  });
+
+  db.users.update({_id: user._id}, {$set: {'newMessages': newNewMessages}});
```

Comment:
not super familiar with mongo syntax -- but didn't think i could unset a specific element in an array?   i see how to unset all of the new messages, or how to set the value of the null message to an empty string.  was reading through this  http://stackoverflow.com/a/19946487